langchain/callbacks/tracers/schemas.py

```diff
@@ -115,7 +119,6 @@ class Run(BaseModel):
     parent_run_id: Optional[UUID]
     reference_example_id: Optional[UUID]
     run_type: RunTypeEnum
-    child_runs: List[Run] = Field(default_factory=list)
```

Comment:
Isnt this still needed for the old tracer?

Reply:
I believe the `Run` object is exclusively used in the V2 tracer right now

Reply:
Oops I was wrong it's used by the n ew one as well still